src/System.Net.Security/src/System/Net/Security/SslState.cs

```diff
@@ -1112,7 +1118,8 @@ private static void PartialFrameCallback(AsyncProtocolRequest asyncRequest)
         //
         private static void ReadFrameCallback(AsyncProtocolRequest asyncRequest)
         {
-            if (NetEventSource.IsEnabled) NetEventSource.Enter(null);
+            if (NetEventSource.IsEnabled)
+                NetEventSource.Enter(null);
```

Comment:
There are a bunch of changes to these one line event source calls.  Any reason?  Across the System.Net libs, we standardized on generally using the single-line syntax, treating the `if (NetEventSource.IsEnabled) ` as a prefixed guard.

Reply:
FYI, there is a setting in the repo that causes that to happen if you ctrl-k,d which I just checked. It's a bit of a pain because if you ever do it (or if it's muscle memory :P due to @davidfowl ) then it breaks all that formatting.